server/doi/api.js

```diff
@@ -14,6 +15,16 @@ const getRequestIds = (req) => {
 	};
 };
 
+const getRequestIdsFromParams = (req) => {
+	const user = req.user || {};
+	return {
+		userId: user.id,
+		communityId: req.query.communityId,
+		collectionId: req.query.collectionId || null,
+		pubId: req.query.pubId || null,
+	};
+};
+
 app.post(
 	'/api/doi/:target',
```

Comment:
What do you think about the way this route is currently written — should `target` be part of the request body rather than the URL for consistency/restfulness?

Reply:
Yeah I was practicing self control 😂 . I can roll that change into this PR!